components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/configure-authentication-flow.jsp

```diff
@@ -228,108 +228,108 @@
-                showAddDefaultSeqConfirm, removeHtmlContent);
-        } else if (showWarn) {
-            $(".err_warn_text").text('Update script with warnings?');
-            showPopupConfirm($(".editor-error-warn-container").html(), "WSO2 Carbon", 250, 550, "OK", "Cancel",
-                showAddDefaultSeqConfirm, removeHtmlContent);
-        } else {
-            showAddDefaultSeqConfirm();
-        }
-    }
-
-    function showAddDefaultSeqConfirm() {
-        showPopupConfirmForDefaultAuthSeq($("#add_default_AuthSeq").html(), "<%=resourceBundle.getString("save.default.seq")%>",
-            250, 550, "<%=resourceBundle.getString("button.save.default.seq")%>",
-            "<%=resourceBundle.getString("button.cancel.default.seq")%>", saveSequence, null);
-    }
-
-    function saveSequence() {
-        var seqDesc = "";
-        $(".sequence-desc").each(function () {
-            if (this.value != "") {
-                seqDesc = $.trim(this.value);
-            }
-        });
-        document.getElementById('seqDesc').value = seqDesc;
-
-        <%
-        String cookie = (String) session.getAttribute(ServerConstants.ADMIN_SERVICE_COOKIE);
-        String backendServerURL = CarbonUIUtil.getServerURL(config.getServletContext(), session);
-        ConfigurationContext configContext = (ConfigurationContext) config.getServletContext()
-                .getAttribute(CarbonConstants.CONFIGURATION_CONTEXT);
-         DefaultAuthenticationSeqMgtServiceClient serviceClient = new DefaultAuthenticationSeqMgtServiceClient(cookie,
-                backendServerURL, configContext);
-         boolean isSeqExists = serviceClient.isExistingDefaultAuthenticationSequence();
-         if (isSeqExists) { %>
-        CARBON.showConfirmationDialog('<%=resourceBundle.getString("alert.confirm.override.default.seq")%>', saveSeq,
-            null);
-        <% } else {
-        %>
-        saveSeq();
-        <%
-        }
-        %>
-    }
-
-    function saveSeq() {
-        $.ajax({
-            type: "POST",
-            url: 'add-as-default-authSeq.jsp?isSeqExists=<%=isSeqExists%>',
-            data: $("#configure-auth-flow-form").serialize(),
-            success: function (data, response, status) {
-                if (data.match("createError") != null) {
-                    CARBON.showErrorDialog('<%=resourceBundle.getString("alert.error.add.default.seq")%>');
-                    return;
-                } else {
-                    CARBON.showInfoDialog('<%=resourceBundle.getString("alert.info.add.default.seq")%>');
-                    return;
-                }
-            },
-            error: function (xhr, ajaxOptions, thrownError) {
-                CARBON.showErrorDialog('<%=resourceBundle.getString("alert.error.add.default.seq")%>' + xhr.status +
-                    thrownError);
-            },
-            async: false
-        });
-    }
-</script>
```

Comment:
Without removing the javascript functions, shall we just remove the usage of each

Reply:
Yes, Done